fitwsarah_frontend/src/views/AdminPanelPage/AdminAppointments.js

```diff
@@ -142,49 +142,83 @@ function AdminAccounts() {
         }
     };
 
+    const handleAcceptedAppointment = (appointmentId) => {
+        const isConfirmed = window.confirm("Are you sure you want to accept this appointment?");
+        if (isConfirmed) {
+            handleAppointmentRequest(appointmentId, 'scheduled');
+        }
+    };
+
 
     const getAppointments = () => {
         const params = new URLSearchParams();
         searchTerm.forEach(term => {
-          if (term[1]) {
-              params.append(term[0], term[1]);
-          }
+            if (term[1]) {
+                params.append(term[0], term[1]);
+            }
         });
 
-        fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/appointments${params.toString() && "?" + params.toString()}`, {
+        fetch(`http://localhost:8080/api/v1/appointments${params.toString() && "?" + params.toString()}`, {
             method: "GET",
             headers: new Headers({
                 Authorization: "Bearer " + accessToken,
                 "Content-Type": "application/json"
             })
         })
-        .then((response) => {
-            if (!response.ok) {
-                throw new Error('Network response was not ok ' + response.statusText);
-            }
-            return response.json();
-        })
-        .then((data) => {
-            console.log(data);
-            setAppointments(data);
-        })
-        .catch((error) => {
-            console.log(error);
-        });
+            .then((response) => {
+                if (!response.ok) {
+                    throw new Error('Network response was not ok ' + response.statusText);
+                }
+                return response.json();
+            })
+            .then((data) => {
+                console.log(data);
+                setAppointments(data);
+            })
+            .catch((error) => {
+                console.log(error);
+            });
     };
 
     const updateAppointmentStatus = (appointmentId, status) => {
-        fetch(`${process.env.REACT_APP_BASE_URL}/api/v1/appointments/${appointmentId}/${status}`, {
+        fetch(`http://localhost:8080/api/v1/appointments/${appointmentId}/cancelled`, {
```

Comment:
You forgot to chanhe the url to ${process.env.REACT_APP_BASE_URL}

Reply:
Oups thanks for reminding me i will change it.

Reply:
Oups thanks for reminding me i will change it.

Reply:
kk